connectors/presto/readme.md

```diff
@@ -1 +1,156 @@
 # IBM created Presto connectors that may be useful to you
+
+## Snowflake connector
+
+### Overview
+
+Snowflake is a cloud-based data platform that allows users to store, analyze, and share data.
+
+The Snowflake connector allows querying and creating tables in an external
+Snowflake database. This can be used to join data between different
+systems like Snowflake and Hive, or between two different Snowflake accounts.
+
+### Connector Configuration
+
+To configure the Snowflake connector, create a catalog properties file in *etc/catalog* named, for example, *snowflake.properties*, to mount the Snowflake connector as the snowflake catalog. Create the file with the following contents, replacing the connection properties as appropriate for your setup:
+
+```
+connector.name=snowflake
+connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}  
+connection-user={username}
+connection-password={password}
+```
+
+The `connection-url` defines the connection information and parameters to pass to the Snowflake JDBC driver.
+
+Syntax: `jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?<connection_params>`
+
+User has to provide only the `account_identifier`, and shouldn't include the prefix `.snowflakecomputing.com`
+
+The `connection-user` and `connection-password` are typically required and determine the user credentials for the connection.
+
+### SSL Configuration
+
+By default, connections to Snowflake use SSL.
+
+### Integrating the Snowflake Plugin to Presto (Local Setup)
+
+1. Clone the repository [watsonx-data](https://github.com/IBM/watsonx-data).
+
+2. Copy the directory *watsonx-data/connectors/presto/presto-snowflake* to the *presto[presto-root]* directory.
```

Comment:
@tdcmeehan - I have included the necessary instructions for utilizing the Snowflake jar and integrating the plugin with Presto here: https://github.com/IBM/watsonx-data/pull/4/files#diff-f41e6bedcbfb5378f732958787ac233f6b8352eb1cca31a7897ee9bdf145779eR81. 

Could you please verify if this is sufficient?

Reply:
It's still not sufficient.  These instructions presume the user will have their own build of Presto, it needs to be rewritten to avoid that entirely.  It should just be as simple as building the JAR for this project and dropping it into the Presto deployment's plugins directory (and adding the configuration).